src/jens/configfile.py

```diff
@@ -28,4 +28,6 @@
 lockdir = string(default='/var/lock/jens')
 [messaging]
 queuedir = string(default='/var/spool/jens-update')
+[sshcommand]
+ssh_cmd_path = string(default='/etc/jens/myssh.sh')
```

Comment:
As discussed, the default should be something that does not force the user to create the wrapper. It should use 'ssh' directly as if GIT_SSH was not set at all.